diff -Nrup libatomic_ops-1.2.orig/src/atomic_ops/sysdeps/gcc/s390.h libatomic_ops-1.2/src/atomic_ops/sysdeps/gcc/s390.h
--- libatomic_ops-1.2.orig/src/atomic_ops/sysdeps/gcc/s390.h	2009-05-20 10:22:59.000000000 +0200
+++ libatomic_ops-1.2/src/atomic_ops/sysdeps/gcc/s390.h	2005-01-11 01:19:06.000000000 +0100
@@ -30,8 +30,8 @@
 /*   may be delayed.  Otherwise everything is ordered.		*/
 /* - There is a hardware compare-and-swap (CS) instruction.	*/
 
-#include "ordered_except_wr.h"
-#include "all_aligned_atomic_load_store.h"
+#include "../ordered_except_wr.h"
+#include "../all_aligned_atomic_load_store.h"
 
 #include "../test_and_set_t_is_ao_t.h"
 /* FIXME: Is there a way to do byte-sized test-and-set?	*/
